code/modules/admin/admin_verbs.dm

```diff
@@ -809,7 +809,17 @@ var/list/admin_verbs_proccall = list (
 		return
 
 	if(!istype(H))
-		return
+		if(istype(H, /mob/living/carbon/brain))

```

Comment:
If you look further down, it won't work for just any brain. It checks if it's an MMI'd brain, then if it's got a body. If it doesn't, it won't do jack.

If I didn't have this, it flat-out wouldn't work for IPCs, morpheus heads or no. It was so wierd.